src/operator/setup.rs

```diff
@@ -184,6 +113,103 @@ where
         Ok(())
     }
 
+    /// List all config map belonging to the cluster. Additional labels to filter or limit
+    /// the configmaps via label selector may be passed via `additional_labels`.
+    pub fn list_config_maps(&self, additional_labels: Vec<String>) -> Vec<ConfigMap> {
+        let mut labels = vec![];
+        if let Some(cluster) = &self.cluster {
+            labels.push(format!(
+                "{}={}",
+                stackable_operator::labels::APP_INSTANCE_LABEL,
+                cluster.name(),
+            ));
+        }
+
+        labels.push(format!(
+            "{}={}",
+            stackable_operator::labels::APP_NAME_LABEL,
+            &self.options.cluster_type
+        ));
+
+        labels.extend(additional_labels);
+
+        self.client
+            .list_labeled::<ConfigMap>(&labels.join(","))
+            .items
+    }
+
+    /// Returns all available nodes in the cluster. Can be used to determine the expected pods
+    /// for tests (depending on the custom resource)
+    pub fn list_nodes(&self) -> Vec<Node> {
+        self.client
+            .list_labeled::<Node>("kubernetes.io/arch=stackable-linux")
+            .items
+    }
+
+    /// List all available pods in the cluster via the name label.
```

Comment:
'Available' is misleading in this context I think. This refers to the existing pods that have been created in kube-apiserver, correct?

Reply:
done